app/core/models/ocelot/LabelCache.scala

```diff
@@ -20,8 +20,9 @@ import core.models.ocelot.stanzas.Stanza
 import play.api.i18n.Lang
 
 trait Flows {
-  def pushFlows(flowNext: Seq[String], continue: String, labelName: Option[String], labelValues: Seq[String], stanzas: Map[String, Stanza]): Labels
-  def takeFlow: Option[(String, Labels)]
+  def pushFlows(flowNext: Seq[String], continue: String, labelName: Option[String], labelValues: Seq[String], stanzas: Map[String, Stanza]): (Option[String], Labels)
```

Comment:
Looking at SequenceStanza I would say that flowNext is a list of strings.

Reply:
Yes, updated